galaxy/values.yaml

```diff
@@ -97,7 +97,7 @@ persistence:
   enabled: true
   name: galaxy-pvc
   # annotations: {}
-  # storageClass:
+  storageClass: nfs
```

Comment:
We don't want a default storageClass so that it uses the cluster default if not set.